src/object/authenticate.c

```diff
@@ -8656,20 +8656,20 @@ au_get_dba_user (void)
 }
 
 /*
- * au_check_serial_authorization - check whether the current user is able to
- *                                 modify serial object or not
+ * au_check_row_authorization - check whether the current user is able to
+ *                                 modify row object or not
  *   return: NO_ERROR if available, otherwise error code
- *   serial_object(in): serial object pointer
+ *   row_object(in): row object pointer
  */
-int
-au_check_serial_authorization (MOP serial_object)
+static int
+au_check_row_authorization (MOP row_object)
```

Comment:
how about **au_check_user_authorization** ?

Reply:
This is a function to check if I have permission to access a specific row and process it.
Whether the emphasis is on row or user.
If it is a user, I feel like I have to check it while passing the user information.

I'm trying to keep the current name unless it's a particularly sensitive issue, is that ok?

Reply:
I think, in common, the word "row" is not used as a function name. It would be better to define it to have a more clear meaning. If you can't find a better name, keep the name.

Reply:
How about au_check_ownership_authorization()?

Reply:
**au_check_ownership_authorization** is not bad. and how about **au_check_owner** more shortly ?

Reply:
OK, then I'll change it to au_check_owner.

Reply:
I think `au_check_user_authorization `or `au_check_authorization` are good.